build-tools/xaprepare/xaprepare/Application/Abi.cs

```diff
@@ -26,12 +26,12 @@ partial class Abi
 			new Abi { Name = AbiNames.HostJit.Win32,           Type = AbiType.HostJit,   Is64Bit = false, IsCross = false, IsHost = true,  IsLlvm = false, IsWindows = true },
 			new Abi { Name = AbiNames.HostJit.Win64,           Type = AbiType.HostJit,   Is64Bit = true,  IsCross = false, IsHost = true,  IsLlvm = false, IsWindows = true },
 
-			new Abi { Name = AbiNames.CrossAot.ArmV7a,         Type = AbiType.CrossAot,  Is64Bit = false, IsCross = true,  IsHost = false, IsLlvm = false, IsWindows = false },
-			new Abi { Name = AbiNames.CrossAot.WinArmV7a,      Type = AbiType.CrossAot,  Is64Bit = false, IsCross = true,  IsHost = false, IsLlvm = false, IsWindows = true },
+			new Abi { Name = AbiNames.CrossAot.ArmV7a,         Type = AbiType.CrossAot,  Is64Bit = true,  IsCross = true,  IsHost = false, IsLlvm = false, IsWindows = false },
```

Comment:
This change (and the others below) is incorrect - the `Is64Bit` field applies to the ABI (the target) itself, not to the *host* tooling (i.e. `ARM v7A` is a 32-bit ABI, not a 64-bit one)

Reply:
Prior to this commit, `make prepare` failed: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2915609&view=logs&s=b9ac06e0-624e-5862-c21d-ce4e0a2cd332&j=95cf21d9-f63e-5a8d-df0a-f986e8a06dc6

```

Running: /usr/local/bin/i686-w64-mingw32-strip "/Users/vsts/agent/2.155.1/work/1/s/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/cross-arm.exe" |  
-- | --
  | stderr \| /usr/local/bin/i686-w64-mingw32-strip:/Users/vsts/agent/2.155.1/work/1/s/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/cross-arm.exe: file format not recognized
Error: Failed to strip the binary file /Users/vsts/agent/2.155.1/work/1/s/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/cross-arm.exe, see logs for error details
```

The above error is fixed by *either* this change, *or* by the removal of the `LlvmRuntime` entry in `Runtimes.cs`.  I don't fully understand `xaprepare`, but it looks like this `Abi` change is what fixed things, because of:

https://github.com/xamarin/xamarin-android/blob/68bb9ae89f898112326d2b93dfb24970cce0a18c/build-tools/xaprepare/xaprepare/Application/MonoCrossRuntime.cs#L23-L26

https://github.com/xamarin/xamarin-android/blob/68bb9ae89f898112326d2b93dfb24970cce0a18c/build-tools/xaprepare/xaprepare/Application/Abi.cs#L108-L109

Additionally, the build *with* this commit *completed* `make prepare`: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2930252&view=results

(It then failed because `llc.exe` wasn't being installed, hence the [subsequent commit](https://github.com/xamarin/xamarin-android/pull/3443/commits/f4db8d4e2439f94c598e27ebb1d502e3771d3632).)

If this change *isn't* the correct change, then different changes -- more "fundamental" changes -- will be required, starting with `Strip` computation within `MonoCrossRuntime`.  Perhaps that should just be hardcoded `KnownProperties.MingwCommandPrefix64`?

```diff
diff --git a/build-tools/xaprepare/xaprepare/Application/MonoCrossRuntime.cs b/build-tools/xaprepare/xaprepare/Application/MonoCrossRuntime.cs
index 0c072c82..c59eb042 100644
--- a/build-tools/xaprepare/xaprepare/Application/MonoCrossRuntime.cs
+++ b/build-tools/xaprepare/xaprepare/Application/MonoCrossRuntime.cs
@@ -18,14 +18,7 @@ namespace Xamarin.Android.Prepare
                                Strip = "strip";
                                StripFlags = "-S";
                        } else if (Context.IsWindowsCrossAotAbi (Name)) {
-                               string mingwPrefix;
-
-                               if (Context.Is32BitCrossAbi (Name))
-                                       mingwPrefix = Path.Combine (Configurables.Paths.MingwBinDir, Context.Properties.GetRequiredValue (KnownProperties.MingwCommandPrefix32));
-                               else
-                                       mingwPrefix = Path.Combine (Configurables.Paths.MingwBinDir, Context.Properties.GetRequiredValue (KnownProperties.MingwCommandPrefix64));
-
-                               Strip = $"{mingwPrefix}-strip";
+                               Strip = Path.Combine (Configurables.Paths.MingwBinDir, Context.Properties.GetRequiredValue (KnownProperties.MingwCommandPrefix64) + "-strip");
                                ExeSuffix = Configurables.Defaults.WindowsExecutableSuffix;
                        } else
                                throw new InvalidOperationException ($"Unsupported cross compiler abi {Name}");
```

Reply:
Hm, you're right, I forgot the cross ABIs are "special". However, since we're dropping 32-bit LLVM tools, the `KnownProperties.MingwCommandPrefix64` change above looks perfectly valid